packages/sample-plugin/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@monorepo/sample-plugin",
+  "version": "0.0.0-fixed",
+  "description": "Sample dynamic plugin project",
+  "private": true,
+  "scripts": {
+    "clean": "rm -rf dist",
+    "build": "yarn clean && NODE_ENV=production yarn webpack",
+    "build-dev": "yarn clean && yarn webpack",
+    "webpack": "node -r ts-node/register $PROJECT_CWD/node_modules/.bin/webpack"
+  },
+  "devDependencies": {
+    "@openshift/dynamic-plugin-sdk": "link:../lib-core/dist",
+    "@openshift/dynamic-plugin-sdk-webpack": "link:../lib-webpack/dist"
```

Comment:
We could use asterix here as well.

Reply:
Check out my comment above, I've tried to explain why linking to `dist` of consumed monorepo packages would make sense, let me know what you think.